packages/rspack/tests/cases/hooks/stillValidModule/rspack.config.js

```diff
@@ -0,0 +1,13 @@
+const MyStillValidModulePlugin = require("./plugins/MyStillValidModulePlugin");
```

Comment:
better use webpack.config.js like other tests, I'm not sure whether out test template will load rspack.config.js

Reply:
got it